client/my-sites/upgrades/checkout-thank-you/jetpack-plan-details.jsx

```diff
@@ -10,19 +10,34 @@ import i18n from 'lib/mixins/i18n';
 import PurchaseDetail from 'components/purchase-detail';
 import userFactory from 'lib/user';
 
+var config = require( 'config' );
+
 const user = userFactory();
 
-const JetpackPlanDetails = () => {
+var JetpackPlanDetails = ( { selectedSite } ) => {
+	const props = {
+		icon: 'cog',
+		title: i18n.translate( 'Set up your VaultPress and Akismet accounts' ),
+		description: i18n.translate(
+			'We emailed you at %(email)s with information for setting up Akismet and VaultPress on your site. Follow the instructions in the email to get started.',
+			{ args: { email: user.get().email } }
+		),
+	};
+
+	if ( config.isEnabled( 'manage/plugins/setup' ) ) {
+		props.title = null;
+		props.description = i18n.translate(
+			'Now we can help you set up and install the Akismet and VaultPress plugins to supercharge your site\'s anti-spam and security capabilities. If you have any questions along the way, we\'re here to help! You can also {{a}}manually install your new plugins{{/a}}.',
+			{ components: {
```

Comment:
How about something like:

"We are ready to automatically install and configure Akismet and VaultPress for your site. In seconds you will be protected from spam and data-loss. If you have any questions along the way, we\'re here to help! You can also perform a manual installation by following {{a}}these instructions{{/a}."

cc @richardmuscat for a copy review.